scoring/src/main/scala/ai/h2o/sparkling/ml/params/HasInputColOnMOJO.scala

```diff
@@ -0,0 +1,14 @@
+package ai.h2o.sparkling.ml.params
+
+import org.apache.spark.ml.param.{Param, Params}
+
+trait HasInputColOnMOJO extends Params {
+  protected val inputCol: Param[String] = new Param[String](this, "inputCol", "Input column name")
+
+  def getInputCol(): String = $(inputCol)
+
+  setDefault(inputCol -> "")
```

Comment:
do we need to set default? or could we set it to a regular value like `inputCol`